### ISSUE TITLE
error, file not found /scratch/software/gmes_linux_64_4.68_07oct2021/parse_set.pl: set.out

### ISSUE THREAD

**Issue by webbchen** [Gaius-Augustus/BRAKER#433]
Dear developers. 
I installed BRAKER yesterday and it's genemark I think which gets stuck.
A test run as follows:
braker.pl --genome genome.fa --prot_seq proteins.fa --bam ../RNAseq.bam --etpmode --softmasking --cores N

yields the following error:
`\# check before the run
\# hard_mask is in the 'auto' mode. hard_mask was set to: 1000
\# creat directories
\# commit input data
\# prepare input data report
\# commit training data
\# prepare training data report
\# prepare initial model
\# find GC of sequence
GC 37
\# build initial ES model
running step ES_A
running gm.hmm on local multi-core system
8 contigs in list
warning on: /scratch/software/gmes_linux_64_4.68_07oct2021/gmhmme3  -m /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_ini/es_ini.mod   -f tr  -d /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1/plus_hmm.gff  -k -0.03  -o /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1/hmmout/dna.fa_1.out  /tmp/annew/BRAKER_test3/GeneMark-ES/data/training/dna.fa_1
warning on: /scratch/software/gmes_linux_64_4.68_07oct2021/gmhmme3  -m /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_ini/es_ini.mod   -f tr  -d /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1/plus_hmm.gff  -k -0.03  -o /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1/hmmout/dna.fa_2.out  /tmp/annew/BRAKER_test3/GeneMark-ES/data/training/dna.fa_2
warning on: /scratch/software/gmes_linux_64_4.68_07oct2021/gmhmme3  -m /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_ini/es_ini.mod   -f tr  -d /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1/plus_hmm.gff  -k -0.03  -o /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1/hmmout/dna.fa_3.out  /tmp/annew/BRAKER_test3/GeneMark-ES/data/training/dna.fa_3
warning on: /scratch/software/gmes_linux_64_4.68_07oct2021/gmhmme3  -m /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_ini/es_ini.mod   -f tr  -d /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1/plus_hmm.gff  -k -0.03  -o /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1/hmmout/dna.fa_4.out  /tmp/annew/BRAKER_test3/GeneMark-ES/data/training/dna.fa_4
warning on: /scratch/software/gmes_linux_64_4.68_07oct2021/gmhmme3  -m /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_ini/es_ini.mod   -f tr  -d /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1/plus_hmm.gff  -k -0.03  -o /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1/hmmout/dna.fa_5.out  /tmp/annew/BRAKER_test3/GeneMark-ES/data/training/dna.fa_5
warning on: /scratch/software/gmes_linux_64_4.68_07oct2021/gmhmme3  -m /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_ini/es_ini.mod   -f tr  -d /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1/plus_hmm.gff  -k -0.03  -o /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1/hmmout/dna.fa_6.out  /tmp/annew/BRAKER_test3/GeneMark-ES/data/training/dna.fa_6
warning on: /scratch/software/gmes_linux_64_4.68_07oct2021/gmhmme3  -m /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_ini/es_ini.mod   -f tr  -d /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1/plus_hmm.gff  -k -0.03  -o /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1/hmmout/dna.fa_7.out  /tmp/annew/BRAKER_test3/GeneMark-ES/data/training/dna.fa_7
warning on: /scratch/software/gmes_linux_64_4.68_07oct2021/gmhmme3  -m /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_ini/es_ini.mod   -f tr  -d /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1/plus_hmm.gff  -k -0.03  -o /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1/hmmout/dna.fa_8.out  /tmp/annew/BRAKER_test3/GeneMark-ES/data/training/dna.fa_8
concatenate predictions: /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1
training level ES_A: /tmp/annew/BRAKER_test3/GeneMark-ES/run/ES_A_1
error, file not found /scratch/software/gmes_linux_64_4.68_07oct2021/parse_set.pl: set.out
error on call: /scratch/software/gmes_linux_64_4.68_07oct2021/parse_set.pl --section ES_A --cfg  /tmp/annew/BRAKER_test3/GeneMark-ES/run.cfg  --v`

I ran the test set after it failed with our data in the same manner.
